backend/src/controllers/webhookController.ts

```
@@ -0,0 +1,96 @@
+            const supabase = getSupabaseAdminClient();
+
+            switch (event.type) {
+                case 'customer.subscription.created':
+                case 'customer.subscription.updated':
+                    const upsertData = {
+                        stripe_subscription_id: subscription.id,
+                        stripe_customer_id: subscription.customer as string,
+                        status: subscription.status as SubscriptionStatus,
+                        current_period_end: new Date((subscription as any)?.current_period_end * 1000 || Date.now()).toISOString(),
+                        updated_at: new Date().toISOString()
+                    };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)